### ISSUE TITLE
Watch after cancel not working

### ISSUE THREAD

**Issue by NassiHarel** [mixer/etcd3#51]
My code:

```js
const { Etcd3 } = require('etcd3');
const uuidv4 = require('uuid/v4');
const path = `path-${uuidv4()}`;
const client = new Etcd3({ hosts: 'http://localhost:4001' });
let watcher;

const watch = async (path) => {
    watcher = await client.watch().prefix(path).create();
}

const unwatch = async () => {
    await watcher.cancel();
}

const test = async () => {
    await watch(path);
    console.log(`success watch`);

    await unwatch();
    console.log(`success unwatch`);

    await watch(path); // this does not resolved
    console.log(`success watch`);
}

test();

```
This test prints:
success watch
success unwatch

I would expect:
success watch
success unwatch
success watch

What I'm doing wrong?

**Comment by NassiHarel** [comment]
Any news?

**Comment by connor4312** [comment]
Holidays are delaying getting this looked at. Should have a chance tomorrow. Thanks for reporting the issue!

**Comment by connor4312** [comment]
Fixed in https://github.com/mixer/etcd3/pull/53

**Comment by connor4312** [comment]
Published in 0.2.7

**Comment by NassiHarel** [comment]
Thank you !
